lib/foreman_maintain/runner.rb

```diff
@@ -6,10 +6,11 @@ class Runner
     require 'foreman_maintain/runner/execution'
     require 'foreman_maintain/runner/stored_execution'
     def initialize(reporter, scenarios, options = {})
-      options.validate_options!(:assumeyes, :whitelist, :force)
+      options.validate_options!(:assumeyes, :whitelist, :force, :rescue)
```

Comment:
I suggest calling it `:rescue_scenario`, rather than just `:rescue` as it's more descriptive name